src/SFA.DAS.ApprenticeFeedback.Data.UnitTests/ApprenticeFeedbackRepository/WhenAddingApprenticeFeedbackTarget.cs

```diff
@@ -35,7 +35,7 @@ public async Task Then_The_ApprenticeFeedbackTarget_Is_Added(ApprenticeFeedbackT
             var result = await _repository.CreateApprenticeFeedbackTarget(apprenticeFeedbackTarget);
 
             //assert
-            result.Value.Should().Be(apprenticeFeedbackTarget.Id);
+            result.Value.Should().Be(apprenticeFeedbackTarget.Id.ToString());
```

Comment:
I think you can drop the .ToString and maybe even use .Value?
or even result.Should().be(apprenticeFeedbackTarget.Id) ?

Reply:
The `.Value` has worked.